### PR TITLE
Mojolicious

### PR DESCRIPTION
**Pull request by dotandimet**
Syntax highlighting for Mojolicious templates http://mojolicio.us

### CODE REVIEW

**Comment by kraih** [comment]
:+1:

**Comment by jberger** [comment]
:+1:

**Comment by firnsy** [comment]
:+1:

**Comment by jhthorsen** [comment]
:+1:

**Comment by isagalaev** [comment]
Oh cool! Let's turn it into a game: every useless "+1" adds one week to the wait for it to be merged. Sounds like fun? :-)

**Comment by kraih** [comment]
@isagalaev Well, if you prefer memes, that can be arranged too. :smiley: But seriously, this looks good and i'd like to see it get merged.

**Comment by isagalaev** [comment]
Every pull request already implies that someone wants it merged. Adding +1s doesn't make it any more probable and just adds useless spam to the maintainers' inboxes. Let's not do this anymore please, someone of us will certainly have a look at it.

**Comment by dotandimet** [comment]
@isagalaev - note that I just squashed all my noisy commits into one, pity I can't do that with the comments.

**Comment by kraih** [comment]
Did we win the game? It has been 5 weeks now.

**Comment by isagalaev** [comment]
There's no time schedule, we're all volunteers here.

**Comment by kraih** [comment]
:beers:

File: src/languages/mojolicious.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
+Language: Perl (Mojolicious)
+Requires: perl.js, xml.js, mojo-ep.js
+Author: Dotan Dimet <dotan@corky.net>
+Description: Perl code with Mojolicious .ep templates embedded in __DATA__ section
+Category: common
+*/
+
+function(hljs) {
```
Comment (Sannis) [inline src/languages/mojolicious.js]: Maybe we can put this straight into perl.js?

Comment (isagalaev) [inline src/languages/mojolicious.js]: Actually, such "bridge" definitions are better. We had the problem with ERB and VBScript within HTML. Both use the same set of delimiters but VBScript was hard-coded as part of HTML and there was simply no way of having ERB too. Now we have erb.js and vbscript-html.js and everything's fine.

**Comment by Sannis** [comment]
Thanks, merged!

**Pull request merged**